features/gezag-minderjarige/bevoegdheid-tot-gezag.feature

```diff
@@ -0,0 +1,112 @@
+  - is overleden
+  - onder curatele staat
+  - is minderjarig
+
+
+  Achtergrond:
+    Gegeven de persoon 'Gerda' met burgerservicenummer '000000012'
+    * is meerderjarig
+    En de persoon 'Aart' met burgerservicenummer '000000024'
+    * is meerderjarig
+    En de persoon 'Ariana' met burgerservicenummer '000000036'
+    * is meerderjarig
+    En de persoon 'Bert' met burgerservicenummer '000000048'
+    * is minderjarig
+
+
+  Regel: Er is sprake van eenhoofdig ouderlijk gezag als één van de ouders met tweehoofdig ouderlijk gezag overleden of niet bevoegd is
+  
+    Scenario: Minderjarige heeft twee ouders en één van de ouders staat onder curatele
+      Gegeven 'Gerda' en 'Aart' zijn getrouwd
+      En persoon 'Bert'
+      * heeft 'Gerda' als ouder
+      * heeft 'Aart' als ouder
+      En 'Gerda' staat onder curatele
+      Als gezag wordt gevraagd met het burgerservicenummer van 'Bert'
+      Dan is het gezag over 'Bert' eenhoofdig ouderlijk gezag met ouder 'Aart'
+
+
+  Regel: Er is tijdelijk geen gezag als beide ouders met tweehoofdig ouderlijk gezag overleden of niet bevoegd zijn
+  
+    Abstract Scenario: Minderjarige heeft twee ouders en beide ouders staan onder curatele
+      Gegeven 'Gerda' en 'Aart' zijn getrouwd
+      En persoon 'Bert'
+      * heeft 'Gerda' als ouder
+      * heeft 'Aart' als ouder
+      En 'Gerda' <bevoegdheid Gerda>
+      En 'Aart' <bevoegdheid Aart>
+      Als gezag wordt gevraagd met het burgerservicenummer van 'Bert'
+      Dan is er tijdelijk geen gezag over 'Bert' met de toelichting '<toelichting>'
+
+      Voorbeelden:
+      | bevoegdheid Gerda    | bevoegdheid Aart     | toelichting                                                                                    |
+      | is overleden         | is overleden         | Tijdelijk geen gezag omdat beide ouders zijn overleden                                         |
+      | staat onder curatele | staat onder curatele | Tijdelijk geen gezag omdat beide ouders onder curate staan                                     |
+      | is minderjarig       | is minderjarig       | Tijdelijk geen gezag omdat beide ouders minderjarig zijn                                       |
+      | is overleden         | staat onder curatele | Tijdelijk geen gezag omdat een ouder overleden is en de andere ouder onder curate staat        |
+      | is overleden         | is minderjarig       | Tijdelijk geen gezag omdat een ouder overleden is en de andere ouder minderjarig is            |
+      | staat onder curatele | is minderjarig       | Tijdelijk geen gezag omdat een ouder onder curatele staat is en de andere ouder minderjarig is |
+  
+
+  Regel: Er is tijdelijk geen gezag als de ouder met eenhoofdig ouderlijk gezag overleden of niet bevoegd is
+  
+    Scenario: Minderjarige is erkend voor 1 januari 2023 en de geboortemoeder is overleden
+      Gegeven 'Bert' is geboren op 30-11-2022
+      * heeft 'Gerda' als ouder op de geboorteakte
```

Comment:
wat is het verschil tussen `heeft 'Gerda' als ouder` en `heeft 'Gerda' als ouder op de geboorteakte`?

Reply:
technisch geen verschil, de stap moet hetzelfde doen. alleen dachten we dat het specifiek in deze feature/rule duidelijker is om expliciet te benoemen dat de ouder op de geboorteakte staat

Reply:
```suggestion
      * heeft 'Gerda' als ouder
```

Reply:
we hebben het hierover gehad. suggestie nemen we niet over. extra tekst is toegevoegd om de relevante situatie te beschrijven. we passen dit wel aan naar "heeft 'Gerda' als ouder vanaf de geboortedatum". De datum van erkenning, dus de datum familierechtelijke betrekking, is relevant hier.